projects/bms_carrier/src/passive_balance.c

```diff
@@ -12,7 +11,6 @@ StatusCode passive_balance(uint16_t *result_arr, size_t len, LtcAfeStorage *afe)
 
   // Iterate through all cells in storage, updating values.
   for (uint8_t i = 0; i < len; i++) {
-    LOG_DEBUG("Value %d: %d\n", i, *result_arr);
     if (*result_arr > cell_voltage_max) {
```

Comment:
Oh lol, I skipped over it, but now that you removed the comment ....
`result_arr` is an array, you can access the elements by doing `result_arr[i]` (`result_arr[0]` in the first case)
incrementing the pointer works but isn't how we would do things typically.